.github/ISSUE_TEMPLATE/question.md

```diff
@@ -23,3 +23,8 @@ Clear and concise details.
  * OS:  name and version
  * ThingsBoard: version
  * Browser: name and version
+
+___________________________________________________________
+**Disclaimer** 
+
+We appreciate your contribution whether it is a bug report, feature request, or pull request with improvement (hopefully). Please comply with the [Community ethics policy](https://opensource.guide/building-community/), and do not expect us to answer your requests immediately. Also, do not treat *GitHub issues* as a support channel.
```

Comment:
The link is referencing a generic web page and not the Community ethics policy

Reply:
Fixed. Changed to a precise Acceptable Use document